optaplanner-website-root/templates/index.ftl

```diff
@@ -12,7 +12,7 @@
         <div class="row">
             <div class="col-md-7">
                 <h1>Solve planning and scheduling problems with OptaPlanner</h1>
-                <p><i>An easy-to-use constraint solver with AI algorithms</i></p>
+                <p class="lead">An easy-to-use, open source constraint solver with AI algorithms</p>
```

Comment:
Having the word "open source" on the front page, above the fold, turns out to be critical :)